public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -227,7 +233,10 @@ export class ConfigEditor extends PureComponent<Props, State> {
           {options.jsonData.authType === 'arn' && (
             <div className="gf-form-inline">
               <div className="gf-form">
-                <InlineFormLabel className="width-14" tooltip="ARN of Assume Role">
+                <InlineFormLabel
+                  className="width-14"
+                  tooltip="ARN of role to assume. The AWS SDK Default credentials-chain will be used to assume it."
+                >
                   Assume Role ARN
```

Comment:
I'm not super familiar with this code so this might be way off, but wouldn't it make sense to use the `label` prop here (if we make the texts the same)?
```suggestion
                  {authProviderOptions[1].label}
```

Reply:
It probably does :sweat_smile: 

I guess it should use the `label` for all options then?

Reply:
I think so as long as the texts align.

Reply:
Oh wait I think I've gotten confused.

This label is for the textbox where you enter the actual ARN of the role to assume. The label of the authentication provider is something different.

You first select that you want Grafana to assume a specific role (`{authProviderOptions[1].label}`) and then you get a new field where you enter which role that is (`Assume Role ARN`).

Maybe it should be changed to something like `ARN of role to assume` but it probably shouldn't match the label of the Authentication Provider.

Reply:
I think this is no longer relevant with the latest commit (dd79792). But there has been new frontend changes that I would be happy if you could review :)

Reply:
@patstrom sure I'll re-review it again on Monday, have a pleasant weekend!